libbindgen/src/ir/enum_ty.rs

```diff
@@ -49,8 +49,8 @@ impl Enum {
         }
 
         let declaration = ty.declaration().canonical();
-        let repr = Item::from_ty(&declaration.enum_type(), None, None, ctx)
-            .ok();
+        let et = &declaration.enum_type().expect("This should be an enum since we checked above!");
```

Comment:
@Wafflespeanut nitpick fixed!

While I've got you here, I've noticed that my changes are breaking the CI build, and I'm not sure why, since `cargo test --features llvm_stable` works for me. Do you have a chance to see what's going on?